controllers/datamovement_controller_lustre.go

```diff
@@ -186,27 +186,31 @@ func (r *DataMovementReconciler) labelStorageNodes(ctx context.Context, dm *nnfv
 }
 
 func (r *DataMovementReconciler) teardownLustreJob(ctx context.Context, dm *nnfv1alpha1.NnfDataMovement) (*ctrl.Result, error) {
-	log := log.FromContext(ctx).WithName("unlabel")
```

Comment:
Should this be `WithName("teardown")`? It does more then unlabel.

Reply:
done